fastlane/lib/fastlane/actions/sync_code_signing.rb

```diff
@@ -59,7 +59,7 @@ def self.description
       end
 
       def self.details
-        "More details https://github.com/fastlane/fastlane/tree/master/match"
+        "More details https://docs.fastlane.tools/actions/match/"
```

Comment:
I'd also change this to `More information:` if you're at it

Reply:
done